src/kafkaConsumer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const Kafka = require('no-kafka');
+const handler = require('./handler');
+const debug = require('debug')('refocus-logging');
+const config = require('./config.env').getConfig();
+
+const clientId = 'consumer-' + process.pid;
+
+debug('The config is', config);
+
+const initConsumer = () => {
+  const consumer = new Kafka.SimpleConsumer({
+    clientId,
+    connectionString: config.connectionString,
+    ssl: {
+      cert: config.sslCert,
+      key: config.sslKey,
+    },
+    maxWaitTime: config.maxWaitTime,
+    maxBytes: config.maxBytes,
+    idleTimeout: config.idleTimeout,
+  });
+
+  consumer.init();
+  debug(`Kafka consumer ${clientId} has been started as ${consumer}`);
+
+  // Construct an object that has a list of all topics as
+  // keys and accordingly you can give it a handler
+  const topicHandlers = config.topics.reduce((obj, topic) => {
+    obj[topic] = (handler) => {
+      try {
+        consumer.subscribe(topic, handler);
+      } catch (err) {
+        // TODO: Not use console.error and still make it testable as of now testing
```

Comment:
I think we have some winston testing in Refocus in case you want to take a look (now or in future).

Reply:
I tried looking for it, but couldn't find it. Got lost in the test directory

Reply:
https://github.com/salesforce/refocus/blob/master/tests/cache/jobQueue/bulkUpsert.js#L113

Reply:
I fixed everything else. But couldn't get this to work, pino is slightly different that winston since the require from pino returns a function that returns an object and the object contains the function that we want to check if its being called. I tried searching the internet a lot, but couldn't get it to work.

Reply:
How about keeping the TODO here as you planned and coming back to this in a later story to try again? Sometimes, you just need a break :)
If it doesn't work at all, do you think we should try using winston instead? Thoughts?

Reply:
Yea for now I was also thinking of keeping it a todo. I am not entirely sure as to what is the best approach in these cases. Winston is supposed to be more stable and testable but Pino is much faster in terms performance and I thought it might be worth using console.error in this place.